e2e/utils.go

```diff
@@ -35,7 +35,7 @@ const (
 	vaultPassphrasePath = "ceph-csi/"
 
 	rookToolBoxPodLabel = "app=rook-ceph-tools"
-	rbdmountOptions     = "mountOptions"
+	rbdMountOptions    = "mountOptions"
```

Comment:
```suggestion
	rbdMountOptions     = "mountOptions"
```

Reply:
To satisfy gofmt, try aligning it with the above assignment

Reply:
it was done while you commenting.. thanks :)